CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 - [#4468](https://github.com/thanos-io/thanos/pull/4468) Rule: Fix temporary rule filename composition issue.
 - [#4476](https://github.com/thanos-io/thanos/pull/4476) UI: fix incorrect html escape sequence used for '>' symbol.
+- [#4519](https://github.com/thanos-io/thanos/pull/4519) Store: fix cannot unmarshal DNS message errors.
```

Comment:
I would probably write `Query: switch to miekgdns DNS resolver as the default one` under the `Changed` section instead :smile: